producer-c/producer-cloudwatch-integ/KvsVideoOnlyStreamingSampleCloudwatch.cpp

```diff
@@ -0,0 +1,228 @@
+#include <com/amazonaws/kinesis/video/cproducer/Include.h>
+#include <com/amazonaws/kinesis/video/utils/Include.h>
+#include "CanaryStreamCallbacks.h"
+
+volatile ATOMIC_BOOL sigCaptureInterrupt;
+
+#define DEFAULT_RETENTION_PERIOD            (2 * HUNDREDS_OF_NANOS_IN_AN_HOUR)
```

Comment:
These should go into the common/include header file

Reply:
Done